infrastructure/application/index.ts

```diff
@@ -658,7 +669,7 @@ export = async () => {
     function createCustomDomain({
       domain,
       name,
-      certificateLocation = "pulumiConfig"
+      certificateLocation = "secretsManager"
```

Comment:
When updating the docs it seemed to make more sense to consider this as the default going forward. It's easier as a step to remove from Pulumi config, and update the `secretLocation` here also.